units/host_monitoring_parent_decommission.go

```diff
@@ -26,9 +27,9 @@ func init() {
 }
 
 type parentDecommissionJob struct {
-	job.Base `bson:"metadata" json:"metadata" yaml:"metadata"`
-	DistroId string `bson:"distro_id" json:"distro_id" yaml:"distro_id"`
-	distro   *distro.Distro
+	job.Base      `bson:"metadata" json:"metadata" yaml:"metadata"`
+	DistroId      string `bson:"distro_id" json:"distro_id" yaml:"distro_id"`
+	MaxContainers int
```

Comment:
This should have a bson tag. This probably will work as is, but there is a trip to the database between when NewParentDecommissionJob is called and when it's run, so it's good to explicitly set how the job serializes